tests/test_queue.py

```diff
@@ -445,36 +457,80 @@ def run_dbos_test_in_process(i: int) -> None:
         },
         "telemetry": {},
         "env": {},
+        "application": {},
     }
     dbos = DBOS(config=dbos_config)
     DBOS.launch()
 
-    Queue("test_queue", worker_concurrency=1)
-    time.sleep(
-        2
-    )  # Give some time for the parent worker to enqueue and for this worker to dequeue
-
+    Queue("test_queue", worker_concurrency=1, concurrency=global_concurrency_limit)
+    start_event.wait()
+    # Signal the parent process we've dequeued
+    start_signal.set()
+    # Wait for the parent process to signal we can move on
+    end_signal.wait()
+    # Complete the task
+    end_event.set()
+
+    # Now whatever is in the queue should be cleared up fast (start/end events are already set)
     queue_entries_are_cleaned_up(dbos)
 
-    DBOS.destroy()
-
 
 def test_worker_concurrency_with_n_dbos_instances(dbos: DBOS) -> None:
-
+    # Ensure children processes do not share global variables (including DBOS instance) with the parent
+    multiprocessing.set_start_method("spawn")
```

Comment:
Found out that GHA uses "fork", which ofc shares the parent's global variable with the child, including the DBOS global instance, which would result in a `DBOS Initialized multiple times with conflicting configuration / fastapi information` error in the children.